src/plugins/console/public/lib/mappings/mapping.test.js

```diff
@@ -1,278 +0,0 @@
-/*
```

Comment:
We've deleted this file, but have we reimplemented or moved the tests anywhere? I'm not seeing them. I want to make sure we're preserving or increasing test coverage.